pyomo/contrib/pynumero/sparse/__init__.py

```diff
@@ -21,13 +21,9 @@
     scipy_available = False
     
 if numpy_available and scipy_available:
```

Comment:
As a matter of Python style, we are (slowly) removing all use of `from foo import *` from Pyomo, in favor of explicit imports (`from foo import bar, baz`)

Reply:
Done. No longer importing `*` in pynumero